.github/workflows/gating.yaml

```diff
@@ -84,3 +84,17 @@ jobs:
         pip install tox
     - name: Test '${{ matrix.tox_env }}' with tox
       run: tox -e ${{ matrix.tox_env }}
+
+  hadolint:
+    name: Hadolint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: hadolint/hadolint-action@v1.5.0
+      with:
+        dockerfile: Containerfile
+        # Ignore list:
+        # * DL3041 - Specify version with dnf install -y <package>-<version>
+        ignore: DL3041
```

Comment:
Do we want to keep ignoring this rule, or do we want to pin the RPM versions in the Containerfile?

Reply:
Hmm, I feel like pinning the RPMs without something like dependabot to update them has more downsides than benefits.

We'll need a process to check for CVEs and manually update the versions of affected RPMs. When bumping Fedora versions, we'll need to change all versions